server/sonar-server/src/main/java/org/sonar/server/qualitygate/QualityGateFinder.java

```diff
@@ -57,18 +59,17 @@ public QualityGateFinder(DbClient dbClient) {
     }
   }
 
+  /**
+   * @deprecated Use {@link #getByOrganizationAndId(DbSession, OrganizationDto, long)}
+   */
+  @Deprecated
   public QualityGateDto getById(DbSession dbSession, long qualityGateId) {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)